modules/customize-certificates-api-add-default.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/api-server.adoc
+
+[id="add-default-api-server-{context}"]
+= Add an API server default certificate
+
+To allow clients outside the cluster to validate the API server's
+certificate, you can replace the default certificate a CA
```

Comment:
"To allow clients outside of the cluster to validate the API server's certificate, you can replace the default certificate with one that is issued by a public or organizational CA."

Reply:
Updated to use this text.